files/metainfo.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.clashcross.clashcross</id>
+
+  <name>ClashCross</name>
+  <summary>Clash's Linux graphical client, based on Flutter.</summary>
```

Comment:
```suggestion
  <summary>Clash's Linux graphical client</summary>
```

You still doesn't say what this does. It doesn't tell what Clash is. Nobody care it's "based on Flutter" beside it being incorrect.